Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,12 @@
 function sum(elements) {
+  let totalSum = 0;
+
+  for (element of elements) {
+    if (typeof element === 'number') {
+      totalSum += element;
+    }
+  }
+  return totalSum;
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```